cli/dcoscli/marathon/main.py

```diff
@@ -761,6 +761,14 @@ def _deployment_list(app_id, json_):
 
     deployments = client.get_deployments(app_id)
 
+    if not deployments:
+        msg = "There are no current deployments"
+        if app_id:
+            msg += " for '{}'".format(app_id)
+        if json_:
```

Comment:
This is the same thing from [pr 242](https://github.com/mesosphere/dcos-cli/pull/242#discussion_r34212844)